### DISCUSSION TITLE
Release planning for PM 3.2.2

### DISCUSSION THREAD

**Discussion by Olf0** [sailfishos-patches/patchmanager#288] [General]
@b100dian, @nephros, when the test-cases (#258 :heavy_check_mark:), #283 (:heavy_check_mark:) and #271 (:heavy_check_mark:) are in, I think we are set to release Patchmanager 3.2.2.

Please object, if you have something in your pipeline, which you want to have merged before that.
As we have no schedule and no serious issues in extant releases (which would have to be fixed ASAP), there is no need to rush to an release (but no reason to be tardy, either).

If not, please test the current release RPM at https://github.com/sailfishos-patches/patchmanager/actions

**Comment by nephros** [comment]
Normally I'd say lets wait for the next SFOS release, but #275 is a bit embarassing, and 4.4 seems to be a bit less SoonNow^TM these days - so go ahead.

Nothing pressing in the pipeline ATM.

**Reply by Olf0** [reply]
> Normally I'd say lets wait for the next SFOS release, but #275 is a bit embarassing, and 4.4 seems to be a bit less SoonNow^TM these days - so go ahead.

This is exactly what I concluded, too.

So I will release between 9 p.m. and 12 p.m. UTC tomorrow (Sunday), if no objections are coming in until then.

**Comment by Olf0** [comment]
Due to PR #289 this became Monday 28 Feb 2022:
- [Releases section](https://github.com/sailfishos-patches/patchmanager/releases/tag/3.2.2)
- [OpenRepos](https://openrepos.net/content/patchmanager/patchmanager), plus new ["Patchmanager Testcases" page](https://openrepos.net/content/patchmanager/patchmanager-testcases) there
- [SailfishOS:Chum testing repo](https://build.sailfishos.org/package/show/sailfishos:chum:testing/patchmanager)
- Submitted to [regular SailfishOS:Chum repo](https://build.sailfishos.org/package/show/sailfishos:chum/patchmanager)
